challenges/unit-testing/katas-tdd/roman-numerals/convert-to-old-roman.js

```diff
@@ -1,3 +1,34 @@
-function convertToOldRoman(n) {}
+const romanNumerals = {
+  one: "I",
+  five: "V",
+  ten: "X",
+  fifty: "L",
+  hundred: "C",
+  fiveHundred: "D",
+  thousand: "M",
+};
+
+function convertToOldRoman(number) {
+  let result = "";
+
+  const numeralValues = [
+    { value: 1000, numeral: romanNumerals.thousand },
+    { value: 500, numeral: romanNumerals.fiveHundred },
+    { value: 100, numeral: romanNumerals.hundred },
+    { value: 50, numeral: romanNumerals.fifty },
+    { value: 10, numeral: romanNumerals.ten },
+    { value: 5, numeral: romanNumerals.five },
+    { value: 1, numeral: romanNumerals.one },
+  ];
+
+  for (let i = 0; i < numeralValues.length; i++) {
```

Comment:
Neat! Notice that this is the same implementation as in the new roman, you just supply it a different numralValues. This is a candidate for a common function

Reply:
I totally agree. I just did it because they have two files in the folder(the old and the new) but the new roman implementation is enough.  Do you think I should just delete the old one?

Reply:
Not delete, in these cases you create a third file, put the function convertToOldRoman there, and the original two files can just initialize the arguments, and call it.
This is a stretch and not required for this module. Your implementation is already very good